packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -129,19 +131,14 @@ library TaikoData {
     /// 10 slots reserved for upgradability, 3 slots used.
     struct Block {
         bytes32 metaHash; // slot 1
-        // blobHash represents the EIP-4844 versioned hash of the blob that
-        // carries the transaction list, or the keccack hash of the RLP-encoded
-        // txList when calldata is used for DA.
-        bytes32 blobHash; // slot 2
-        address assignedProver; // slot 3
+        address assignedProver; // slot 2
         uint96 livenessBond;
-        uint64 blockId; // slot 4
+        uint64 blockId; // slot 3
         uint64 proposedAt;
         uint32 nextTransitionId;
         uint32 verifiedTransitionId;
         uint16 minTier;
```

Comment:
minTier can also be moved MetaData as it is not used during verification. If you do so, you can also remove `minTier` from Event BlockProposed together with `blobHash` and `blobUsed`